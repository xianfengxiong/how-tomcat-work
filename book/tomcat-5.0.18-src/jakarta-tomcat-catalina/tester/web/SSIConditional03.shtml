Before if block.
<!--#set var="test" value="unquoted multi-word value" -->
<!--#if expr="\"$test\" = unquoted multi-word value" -->
test = "unquoted multi-word value"
<!--#else -->
test = not "unquoted multi-word value"
<!--#endif -->
After if block.
