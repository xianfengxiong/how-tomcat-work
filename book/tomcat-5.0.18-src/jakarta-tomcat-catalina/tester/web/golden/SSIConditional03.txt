Before if block.


test = "unquoted multi-word value"

After if block.
